### PR TITLE
Created Variable to Allow Right Text Padding to be adjusted.

### PR DESCRIPTION
**Pull request by ColinEHiggins**
Added the variable `textPaddingRight` to the `addMenuWords` function in MenuGraphr.ts and to the `IMenuBase` interface in IMenuGraphr.ts.
Fixed the white space deletion.
Properly formatted changes.
Ran `gulp setup` and `npm update`.

### CODE REVIEW

File: src/MenuGraphr.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class MenuGraphr implements IMenuGraphr {
 
     /**
      * Initializes a new instance of the MenuGraphr class.
-     * 
+     *
```
Comment (JoshuaKGoldberg) [inline src/MenuGraphr.ts]: I'll need you to undo the whitespace changes.

File: src/MenuGraphr.ts [hunk 2]
```diff
@@ -994,7 +994,7 @@ export class MenuGraphr implements IMenuGraphr {
         }
 
         // If the next word would pass the edge of the menu, move down a line
-        if (x + this.computeFutureWordLength(words[i + 1], textWidth, textPaddingX) >= menu.right - menu.textXOffset) {
+        if (x + this.computeFutureWordLength(words[i + 1], textWidth, textPaddingX) >= menu.right - menu.textXOffset - this.GameStarter.unitsize * 2) {
```
Comment (JoshuaKGoldberg) [inline src/MenuGraphr.ts]: >  this.GameStarter.unitsize * 2

This should be a configurable amount. Not every menu will want it to be `unitsize * 2`.

File: src/MenuGraphr.ts [hunk 3]
```diff
@@ -945,7 +946,8 @@ export class MenuGraphr implements IMenuGraphr {
 
         textSpeed = menu.textSpeed;
         textWidth = (menu.textWidth || textProperties.width) * this.GameStarter.unitsize;
-        textPaddingX = (menu.textPaddingX || textProperties.paddingX) * this.GameStarter.unitsize;
```
Comment (JoshuaKGoldberg) [inline src/MenuGraphr.ts]: Why get rid of `textProperties.paddingX` for `textPaddingX`?

Comment (ColinEHiggins) [inline src/MenuGraphr.ts]: Oh my bad I changed the wrong line.

**Comment by JoshuaKGoldberg** [comment]
A note for future readers: the extra files under `test` and `typing` are included because `gulp-shenanigans` was upgraded. This is good.

File: src/MenuGraphr.ts [hunk 4]
```diff
@@ -992,9 +994,8 @@ export class MenuGraphr implements IMenuGraphr {
 
             return things;
         }
-
```
Comment (JoshuaKGoldberg) [inline src/MenuGraphr.ts]: Why remove this blank line? I rather liked it.

**Pull request merged**